drivers/modem/Kconfig.ublox-sara-r4

```diff
@@ -112,4 +112,18 @@ config MODEM_UBLOX_SARA_R4_NET_STATUS_PIN
 
 endif # MODEM_UBLOX_SARA_R4_NET_STATUS
 
+config MODEM_UBLOX_SARA_RSSI_WORK
+	bool "Enable RSSI polling work"
+	default n
```

Comment:
No need to have `default n` here as the options are off by default.

Reply:
changed to `default y` as previous behavior was enabling this feature.